### ISSUE TITLE
Non-Veteran Claimants | Research ability to intake representatives

### ISSUE THREAD

**Issue by rutvigupta-design** [department-of-veterans-affairs/caseflow#15382]
### User story

As a BVA Intake users, I need a way to manually enter POA information if I am adding a new non-Veteran claimant, so that they can be included in any outgoing correspondence.

### Problem statement

During discovery research, the need to enter in POA information came up but the team still needs to understand what information needs to be collected, and what this information is used for down the line. 

### What is out of scope?

- Designs to intake this information
- Designs to edit this information

### Background/context

Before we can design the ability to intake POA information, we need to understand what information we need to design for, and where this will be used. 

### What are the unknowns?

- Should BVA users add any POA (accredited) to SHARE/VBMS instead of Caseflow? 
- What needs to happen with unlicensed POA (e.g. a family member)
- What happens if there are multiple POAs involved?

### What questions do we hope to answer? What are our learning goals?

- Who is a POA? Is this used interchangeable with "representative"? 
- Would everyone have a POA? Do they need to? 
- What are the scenarios for having multiple POAs? 
- How would an Intake user gather POA information? What all are they looking at or checking?
- If someone hasn't listed out a POA, should the appeal still be intaken?
- Is it correct to assume that most POAs are likely accredited attorneys? What happens if a family member, or an unlicensed individual has POA?
- What information is critical to notify the POA? Anything other than name and address?
- Where should a non-Veterans POA info be displayed? Who would use this post-Intake?
- What if there are multiple POAs? What needs to happen? Would a user need to see all this information within Caseflow? Would all POAs always be active?
- Post-intake, what are the touchpoints where a POA would need to be notified?
- What notification letters are sent out once an appeal has been intaken? What is used to create these? Can addresses be manually entered or edited?

### Existing design and content

![Screen Shot 2020-09-24 at 9.57.55 AM.png](https://images.zenhubusercontent.com/5d07e8c541e1974be7393f74/bff51b36-be45-46d3-becd-330a70cfba22)
This is where information about non-Veteran appellants is currently displayed.

### Success criteria

- High enough level of confidence to continue designs

### Technical/logistical constraints (if known)

- Scheduling time with appropriate users to answer these questions
- Coordination with VBA to answer open q's and understand the process of adding claimants to VBMS or SHARE

**Comment by leikkisa** [comment]
Closing this - moved to Jira (and completed!)
